package.json

```diff
@@ -10,7 +10,7 @@
   },
   "private": true,
   "dependencies": {
-    "@apollo/client": "^3.0.0-beta.50",
+    "@apollo/client": "^3.2.5",
```

Comment:
aren't we not using this anymore? in general i would like to not introduce any new libraries we won't be using and removing any existing ones we will stop using

Reply:
also if we're sure this won't be used anymore, i can set up a different ticket to migrate all our existing usages to an older version or to offix entirely (cc @wongcmichelle)